Source/UI/NewUI/PaymentForm/View/PaymentHeaderView.swift

```diff
@@ -0,0 +1,114 @@
+//
+//  PaymentHeaderView.swift
+//  Frames
+//
+//  Created by Ehab Alsharkawy
+//  Copyright © 2022 Checkout. All rights reserved.
+//
+
+import UIKit
+
+public final class PaymentHeaderView: UIView {
+  private var style: PaymentHeaderCellStyle?
+
+  private var stackView: UIStackView = {
+    let view = UIStackView().disabledAutoresizingIntoConstraints()
+    view.axis = .vertical
+    view.spacing = 10
+    return view
+  }()
+
+  private lazy var headerLabel: LabelView = {
+    LabelView()
+  }()
+
+  private lazy var subtitleLabel: LabelView = {
+    LabelView()
+  }()
+
+  private(set) lazy var iconsViewContainer: UIView = {
+    UIView()
+  }()
+
+  private(set) lazy var iconsStackView: UIStackView = {
+    let view = UIStackView().disabledAutoresizingIntoConstraints()
+    view.axis = .horizontal
+    view.alignment = .leading
+    view.distribution = .fillEqually
+    view.spacing = Constants.Padding.xs.rawValue
+    return view
+  }()
+
+  override init(frame: CGRect) {
+    super.init(frame: frame)
+    setupViewsInOrder()
+  }
+
+  required init?(coder: NSCoder) {
+    fatalError("init(coder:) has not been implemented")
+  }
+
+  func update(style: PaymentHeaderCellStyle?) {
+    guard let style = style else { return }
+    self.style = style
+
+    backgroundColor = style.backgroundColor
+    headerLabel.isHidden = style.headerLabel == nil
+    subtitleLabel.isHidden = style.subtitleLabel == nil
+
+    if let headerStyle = style.headerLabel {
+      headerLabel.update(with: headerStyle)
+    }
+    if let subtitleStyle = style.subtitleLabel {
+      subtitleLabel.update(with: subtitleStyle)
+    }
+
+    updateIconsStack()
+  }
+
+  private func updateIconsStack() {
+    guard let icons = style?.schemaIcons else {
+      iconsStackView.isHidden = true
+      return
+    }
+    iconsStackView.isHidden = false
```

Comment:
At the call site you are already unwrapping the style. Why not pass it as a parameter and save the unwrapping? This could become
```swift
private func updateIconsStack(with icons: [UIImage] {
  // Create imageViews 
```